.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - GITHUB_TOKEN=foo
```

Comment:
An encrypted, actual GitHub token should be added here

Run

```
$ travis encrypt GITHUB_TOKEN=...
```

and then replace this

```diff
-    - GITHUB_TOKEN=foo
+    - secure: ...
```

💁 This helps with preventing issues regarding GitHub API rate limits, when installing dependencies with `composer`.